### ISSUE TITLE
Installing on new computer

### ISSUE THREAD

**Issue by landreaus** [rmcrackan/Libation#593]
**Describe the bug**
I try to run Libation 10.2.1 and I get a pop up box telling me to come here and report the problem.

**To Reproduce**

Try to open Libation on my new computer

**Expected behavior**
I expected the program to open

**Platform**

Windows 11

**Log Files**
[_Log202304.log](https://github.com/rmcrackan/Libation/files/11312737/_Log202304.log)

[Log202304.log](https://github.com/rmcrackan/Libation/files/11312738/Log202304.log)

**Comment by rmcrackan** [comment]
It's a problem with the LibationFiles location `.;LibationFiles`. Once this points to a valid location the error will go away.

LibationFiles is injected into the db connection string to show where the database file lives. The connection string uses `;` to separate fields. So injecting one this way breaks it.

**Comment by rmcrackan** [comment]
Let's see if I can do your fancy embedded code trick:

https://github.com/rmcrackan/Libation/blob/429367d21c8ba6e748cfcebb30c99586175980e8/Source/LibationFileManager/SqliteStorage.cs#L8-L9

**Comment by landreaus** [comment]
That works. But I never touched the appsettings file. When I installed it, though, the program asked me if I was a new user and I answered no, because I'm not. And then it asked where my books were stored, and I told it. Based on that, I think the program customized the appsettings file.

**Comment by landreaus** [comment]
You said: It's a problem with the LibationFiles location .;LibationFiles. Once this points to a valid location the error will go away. LibationFiles is injected into the db connection string to show where the database file lives. The connection string uses ; to separate fields. So injecting one this way breaks it.

Once again, I didn't do that. That's how the program named that file on my old computer. I simply moved it, without editing anything, over to the new computer. It worked with that ; on the old computer, and once I deleted the app settings file, it is working fine on this one.
